packages/shopping-cart/src/types.ts

```diff
@@ -693,6 +693,7 @@ export interface RequestCartProductExtra extends ResponseCartProductExtra {
 	 *
 	 */
 	hosting_intent?: string;
+	flow_name?: string;
```

Comment:
If you do end up going this route, can you add a comment above this property? Eventually we're going to validate all of these properties and I'd like to make sure they're all documented.

For example, it's not clear how `flow_name` differs from `signup_flow` above?

Reply:
I missed `signup_flow`. I originally had `is_migration_flow`, which felt way too specific, so I renamed it just before publishing this PR. Thanks, @sirbrillig.

I hope to avoid a front-end PR. Do you know of a way to get the flow name in the checkout code ( see D161970-code) without explicitly sending it? The fact that `signup_flow` exists makes me think we need to send it from the front end.

Reply:
> Do you know if the flow name is available on the backend already? 

It looks like it's used for the `WPCOM_Billing_Payment->get_signup_flow()` function which is used for some Marketplace things (see the code where Marketplace products are activated).